ipymd/__init__.py

```diff
@@ -2,3 +2,6 @@
 from .core.format_manager import convert, format_manager
 from .core.contents_manager import IPymdContentsManager
 from .core.scripts import convert_files
+
+
+__version__ = '0.1.0-dev'
```

Comment:
i think this needs to be 0.1.0.dev0 https://www.python.org/dev/peps/pep-0440/#implicit-development-release-number